sql-gremlin/src/test/java/org/twilmes/sql/gremlin/adapter/GremlinSqlAdvancedSelectTest.java

```diff
@@ -144,6 +147,14 @@ public void testWhere() throws SQLException {
                 rows(r("Tom", 35), r("Juanita", 50)));
     }
 
+    // TODO: Support NOT on operators
+    @Test
+    @Disabled
```

Comment:
Rebase and try this again. The pull request I merged after you opened this added support for this in SELECT, it might have propagated here.